docs/output.md

```diff
@@ -28,6 +29,14 @@ Tabix generated index files with `.tbi` extension for all `(g).vcf` files that a
 
 The GATK GenotypeGVCFs module translates genotype (g) vcf files into classic vcf files. The key difference between a regular VCF and a GVCF is that the GVCF has records for all sites, whether there is a variant call there or not.
 
+### Filter VCFs
+
+VEP annotated VCF files can be filtered for certain flags present after VEP annotation. Notably, this enables filtering for variants with certain impact levels or consequences. Examples include:
+
+- Setting `--filter 'INFO/CSQ ~ "HIGH"'` will keep only variants with the IMPACT marked as high.
+- Setting `--filter 'INFO/CSQ ~ "MODERATE" || INFO/CSQ ~ "HIGH"'` will keep only variants with the IMPACT marked as moderate or high.
+- Setting `--filter 'INFO/CSQ ~ "missense_variant"'` will keep only variants with the consequence `missense_variant`.
+
```

Comment:
This description I would have expected in usage.md, and here a list of files, if any